include/rospack/rospack.h

```diff
@@ -146,6 +146,7 @@ class ROSPACK_DECL Rosstackage
     bool crawled_;
     std::string name_;
     std::string tag_;
+    std::vector<std::string> licenses_;
```

Comment:
Removed this line. I can't remember, but I think this was added simply by mistake.